app/views/layouts/application.html.haml

```diff
@@ -35,9 +35,10 @@
     %a.govuk-skip-link{ href: '#main-content' }
       = t('layouts.skip_content')
 
-    - if GoogleAnalytics::DataTracking.tag_manager? && !@disable_analytics
-      %noscript
-        %iframe{ src: "https://www.googletagmanager.com/ns.html?id=#{ENV['GTM_TRACKED_ID']}", style: 'display: none; visibility:hidden', height: '0', width: '0' }
+    - if GoogleAnalytics::DataTracking.enabled? && !@disable_analytics
+      - if GoogleAnalytics::DataTracking.tag_manager?
```

Comment:
`tag_manager?` already includes `enabled?`. Is there a reason why this needs to be broken into two lines?